controllers/grid/notifications/TaskNotificationsGridHandler.inc.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @file controllers/grid/notifications/TaskNotificationsGridHandler.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class TaskNotificationsGridHandler
+ * @ingroup controllers_grid_notifications
+ *
+ * @brief Handle the display of task notifications for a given user
+ */
+
+// Import UI base classes.
+import('lib.pkp.controllers.grid.notifications.NotificationsGridHandler');
+
+class TaskNotificationsGridHandler extends NotificationsGridHandler {
+
+	/**
+	 * Constructor
+	 */
+	function TaskNotificationsGridHandler() {
+		parent::NotificationsGridHandler();
+	}
+
+	/**
+	 * @see GridHandler::loadData()
```

Comment:
`@see` doesn't import parameter descriptions; `@copydoc` does. When using `@see`, you'll need to still describe the parameters.